kimchi/src/circuits/gate.rs

```diff
@@ -107,8 +107,9 @@ pub enum GateType {
     /// Range check (16-24)
     RangeCheck0 = 16,
     RangeCheck1 = 17,
-    // ForeignFieldAdd = 25,
-    // ForeignFieldMul = 26,
+    ForeignFieldAdd = 25,
+    //ForeignFieldMul = 26,
+    ForeignFieldFin = 27,
```

Comment:
There's no reasons to shorten names in this case.  For consistency call it `ForeignFieldFinal` like `ChaChaFinal`